roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -47,7 +47,7 @@ bool ControlUtils::pointInRectangle(Vector2 PointToCheck, Vector2 SP1, Vector2 S
     if (pointInTriangle(PointToCheck, SP1, SP2, SP3)) {
         return true;
     }
-    else return pointInTriangle(PointToCheck, SP4, SP1, SP2);
+    else return pointInTriangle(PointToCheck, SP4, SP1, SP3);
```

Comment:
why is this changed? was this a mistake?

Reply:
See commit message, this was a very very ugly bug (it was not actually checking a rectangle).